addons/stock/models/stock_picking.py

```diff
@@ -820,28 +819,38 @@ def button_validate(self):
         if sms_confirmation:
             return sms_confirmation
 
-        if no_quantities_done:
-            view = self.env.ref('stock.view_immediate_transfer')
-            wiz = self.env['stock.immediate.transfer'].create({'pick_ids': [(4, self.id)]})
-            return {
-                'name': _('Immediate Transfer?'),
-                'type': 'ir.actions.act_window',
-                'view_mode': 'form',
-                'res_model': 'stock.immediate.transfer',
-                'views': [(view.id, 'form')],
-                'view_id': view.id,
-                'target': 'new',
-                'res_id': wiz.id,
-                'context': self.env.context,
-            }
+        # Run the pre-validation wizards. Processing a pre-validation wizard should work on the
```

Comment:
care with the sanity check with multiple picking that have different picking type (server action for e.g.)

Reply:
fixed